### ISSUE TITLE
Increase gap size also modify item height 

### ISSUE THREAD

**Issue by vupham30** [katoid/angular-grid-layout#125]
When I set [gap] as 0, the size of item is corrected, for example item with h as 50 and [rowHeight] as 10 will have height as 510. But if I increase [gap], the height are not corrected any more.

**Comment by llorenspujol** [comment]
Hi @vupham30 !

I don't know if I understood correctly, but take into account that having a gap can increase the total height of the grid item, and this is how it is intended to work. Here is an example:

h=50, rowHeight=10, gap=0 -> Total height would be 500px
h=50, rowHeight=10, gap=1 -> Total height would be 549px (500 + ((cols-1) * gap)

**Comment by vupham30** [comment]
yes, that's the issue, I think the gap should not be counted into the size of the item.
